samples/snippets/src/main/java/com/example/speech/SpeechTranscribeMultiRegion.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.speech;
+
+// [START speech_transcribe_with_multi_region_gcs]
```

Comment:
I don't see how this sample demonstrates the multi-region aspect of transcribing audio files store in GCS. There seems to be only 1 region. If you feel this sample is demonstrating it correctly, please add in comments to explain. Otherwise, please expand the sample add/or update the region tag to appropriately reflect what the sample does.

Reply:
Did not intend to change this file.